ViAn/Video/shapes/shape.h

```diff
@@ -20,7 +20,8 @@ class Shape {
     static cv::Scalar qcolor_to_scalar(QColor col);
     static cv::Point qpoint_to_point(QPoint pnt);
 
-    static const int LINE_THICKNESS = 2;
+    static const int LINE_THICKNESS = 2; // Constant used for the thickness of the drawn shapes.
+    static constexpr double ALPHA = 0.6; // Costant used for the opacity.
 
 protected:
     cv::Scalar colour;
```

Comment:
Since we use the british spelling for analyse/analyze, colour is technically correct if we wanna be consistent in that regard, however just 7 lines above you use a QColor which use the american spelling and this is the spelling used for most of the code. So we should probably be consistent here and just use one of the spellings for color/colour. I personally don't like the u so I vote for using color. If that means that we should change to analyze I don't know.